app/src/main/kotlin/io/imhungry/model/RestaurantTagEntity.kt

```diff
@@ -0,0 +1,11 @@
+package io.imhungry.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+
+@Entity(tableName = "restaurant_tag",
+    primaryKeys = ["tag", "restaurant_id"])
```

Comment:
For consistency, you should define the primary key using @PrimaryKey instead of the entity parameter.